### DISCUSSION TITLE
lspsaga Breadcrumbs is not showing on the buffer

### DISCUSSION THREAD

**Discussion by convers39** [NvChad/NvChad#2770] [Q&A]
https://nvimdev.github.io/lspsaga/breadcrumbs/

<img width="401" alt="image" src="https://github.com/NvChad/NvChad/assets/42597395/76d5f984-c9f0-4926-9d98-9868e960e1ae">

the Breadcrumbs is enabled by default but not showing on the top of the buffer.

I thought it was conflicting with Tabufline and then tried to disable Tabufline but didn't work. 

Not sure if anyone has a similar issue.

**Comment by siduck** [comment]
@convers39 show your code and please ask on discord server as its  more active than this github discussions as you get quick reply there
